public/styles/components/_input-radio.scss

```diff
@@ -1,14 +1,6 @@
 $outer-circle-diameter: $ffs-body-assertive;
 $inner-circle-diameter: space(md);
 
-.bsds-fieldset {
```

Comment:
did we test the different styles, incl the button group style?

Reply:
The utility class keeps the spacing in both regular radio and button group